apps/api/serializers/metrics.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework import serializers
```

Comment:
General question here, do we have permission checks on this? (I think we do but just double checking)

Reply:
There are permissions checks in the views. Is that what you mean? I have tests written for them as well.

Reply:
I meant permission classes that we use to do checks on the views.